go-controller/pkg/ovn/egressip_test.go

```diff
@@ -5094,8 +5094,15 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 					EgressIPs: []string{
 						updatedEgressIP.String(),
 					},
+					// PodSelector: metav1.LabelSelector{
+					// 	MatchLabels: egressPodLabel,
+					// },
+					// https://issues.redhat.com/browse/OCPBUGS-24055
 					PodSelector: metav1.LabelSelector{
-						MatchLabels: egressPodLabel,
+						MatchExpressions: []metav1.LabelSelectorRequirement{{
+							Key:      "someNonExistingKey",
+							Operator: metav1.LabelSelectorOpDoesNotExist,
```

Comment:
let's also do a test for `Key` `NotIn` `Values being a string of stuff`

Reply:
good idea. ;)